<?xml version='1.0' encoding='utf-8'?>
<ui version="4.0">
  <!-- Generated for PYSIDE6 -->
  <class>ConnectionTest</class>
  <widget class="QDialog" name="ConnectionTest">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>ConnectionTest</string>
    </property>
    <widget class="QPushButton" name="okButton">
      <property name="text">
        <string>OK</string>
      </property>
      <property name="geometry">
        <rect>
          <x>200</x>
          <y>220</y>
          <width>100</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="extraLabel">
      <property name="text">
        <string>Extra Widget</string>
      </property>
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>100</y>
          <width>200</width>
          <height>25</height>
        </rect>
      </property>
    </widget>
    <widget class="QPushButton" name="cancelButton">
      <property name="text">
        <string>Cancel</string>
      </property>
      <property name="geometry">
        <rect>
          <x>50</x>
          <y>220</y>
          <width>100</width>
          <height>40</height>
        </rect>
      </property>
    </widget>
  </widget>
  <connections>
    <connection>
      <sender>okButton</sender>
      <signal>clicked()</signal>
      <receiver>ConnectionTest</receiver>
      <slot>accept()</slot>
    </connection>
  </connections>
</ui>
